### DISCUSSION TITLE
Nextjs not installing

### DISCUSSION THREAD

**Discussion by Natus05** [vercel/next.js#53278] [Help]
### Summary

I've been repeatedly getting and error message while trying to install nextjs/reactjs
![image](https://github.com/vercel/next.js/assets/110559496/2d89a761-d954-4a20-927d-cba3396551a7)
![error](https://github.com/vercel/next.js/assets/110559496/9e549fce-e61a-4479-90bf-9e7c7ace70ee)

### Additional information

```js
npm ERR! code ENOENT
npm ERR! syscall lstat
npm ERR! path C:\Users\ADMIN\AppData\Roaming\npm
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, lstat 'C:\Users\ADMIN\AppData\Roaming\npm'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in: C:\Users\ADMIN\AppData\Local\npm-cache\_logs\2023-07-28T09_16_23_006Z-debug-0.log
```

**Comment by ParamThakkar123** [comment]
Try removing the --example link. Use only npx create-next-app nextjs-blog. Actually it's not able to get the files from the url you mentioned. For this reason it is throwing the error.

**Comment by icyJoseph** [comment]
What Node.js version do you have?
